test/integration/openingTimes.js

```diff
@@ -64,7 +64,7 @@ describe('app', () => {
           const $ = cheerio.load(res.text);
 
           expect($('h2.opening-times').first().text().trim()).to.include('Reception opening times');
-          expect($('h2.opening-times').last().text().trim()).to.include('GP appointment times');
+          expect($('h2.opening-times').last().text().trim()).to.include('When GPs see patients');
```

Comment:
Based on the history it seems that at some point it only covered part of the message which could happen again. It might be a matter of style.